mysql/const.go

```diff
@@ -172,6 +172,7 @@ const (
 
 const (
 	DEFAULT_ADDR                  = "127.0.0.1:3306"
+	DEFAULT_IPv6_ADDR             = "[::1]:3306"
```

Comment:
```suggestion
	DEFAULT_IPV4_ADDR                  = "127.0.0.1:3306"
	DEFAULT_IPV6_ADDR             = "[::1]:3306"
```

Not sure if we should uppercase the `V` here or not..

Reply:
Changing `DEFAULT_ADDR` to `DEFAULT_IPv6_ADDR` might break compatibility, maybe it's better to leave it unchanged.